ant/src/initialize.lua

```diff
@@ -0,0 +1,20 @@
+local initialize = {}
+
+function initialize.initialize(state)
+	local name, ticker, balances, controllers, records =
+		state.name, state.ticker, state.balances, state.controllers, state.records
+
+	assert(type(name) == "string", "name must be a string")
+	assert(type(ticker) == "string", "ticker must be a string")
+	assert(type(balances) == "table", "balances must be a table")
+	assert(type(controllers) == "table", "controllers must be a table")
```

Comment:
todo: check each object in the controllers

for _, address in ipairs(controllers) then
    assert(utils.validateArweaveId(address), "Controller must be a valid arweave ID")
end

Reply:
Added some basic validation that should cover this